app/helpers/admin/organizations_helper.rb

```diff
@@ -0,0 +1,11 @@
+module Admin
+  module OrganizationsHelper
+    def upcoming_conferences_count(organization)
+      organization.conferences.where('end_date >= ?', Date.current).count
```

Comment:
This condition (`.where('end_date >= ?', Date.current)`) can be made into a named scope in the Conference model:

```ruby
scope :upcoming, -> { where('end_date >= ?', Date.current) }
```

Then, you can do `organization.conferences.upcoming.count`, and I would just put that directly in the HAML, no need for a helper.